pkg/db/matviews.go

```diff
@@ -239,27 +272,25 @@ GROUP BY
     tests.id, tests.name, jira_components.name, jira_components.id, suites.name, open_bugs.open_bugs, prow_jobs.variants, prow_jobs.release
 `
 
-const testAnalysisByVariantMatView = `
+const testAnalysisByVariantView = `
 SELECT
-    tests.id AS test_id,
-    tests.name AS test_name,
-    tests.watchlist,
-    date(prow_job_runs."timestamp") AS date,
-    unnest(prow_jobs.variants) AS variant,
-    prow_jobs.release,
-    COUNT(*) FILTER (WHERE prow_job_runs."timestamp" >= (|||TIMENOW||| - '14 days'::interval) AND prow_job_runs."timestamp" <= |||TIMENOW|||) AS runs,
-    COUNT(*) FILTER (WHERE prow_job_run_tests.status = 1 AND prow_job_runs."timestamp" >= (|||TIMENOW||| - '14 days'::interval) AND prow_job_runs."timestamp" <= |||TIMENOW|||) AS passes,
-    COUNT(*) FILTER (WHERE prow_job_run_tests.status = 13 AND prow_job_runs."timestamp" >= (|||TIMENOW||| - '14 days'::interval) AND prow_job_runs."timestamp" <= |||TIMENOW|||) AS flakes,
-    COUNT(*) FILTER (WHERE prow_job_run_tests.status = 12 AND prow_job_runs."timestamp" >= (|||TIMENOW||| - '14 days'::interval) AND prow_job_runs."timestamp" <= |||TIMENOW|||) AS failures
+	byjob.test_id AS test_id,
+	byjob.test_name AS test_name,
+	byjob.date AS date,
+	unnest(prow_jobs.variants) AS variant,
+	prow_jobs.release,
+	SUM(runs) as runs,
+	SUM(passes) as passes,
+	SUM(flakes) as flakes,
+	SUM(failures) as failures
 FROM
-    prow_job_run_tests
-    JOIN tests ON tests.id = prow_job_run_tests.test_id
-    JOIN prow_job_runs ON prow_job_runs.id = prow_job_run_tests.prow_job_run_id
-    JOIN prow_jobs ON prow_jobs.id = prow_job_runs.prow_job_id
+	prow_test_analysis_by_job_14d_matview byjob
+	JOIN tests ON tests.id = byjob.test_id
+	JOIN prow_jobs ON prow_jobs.name = byjob.job_name
 WHERE
-    prow_job_run_tests.created_at > (|||TIMENOW||| - '14 days'::interval) AND prow_job_runs."timestamp" > (|||TIMENOW||| - '14 days'::interval)
+	byjob.date >= (now() - '15 days'::interval)
```

Comment:
I don't think we really use the `pinned date` stuff but since you have [syncPostgresViews](https://github.com/openshift/sippy/pull/2069/files#diff-7fa1b9fbef139e2c0d9eadf51d97fa898c9360bcfcbdcc407207629fcdca60cfR131) should you use `|||TIMENOW|||` instead of now?